nixos/modules/services/misc/freeswitch.nix

```diff
@@ -0,0 +1,102 @@
+{ config, lib, pkgs, ...}:
+with lib;
+let
+  cfg = config.services.freeswitch;
+  pkg = cfg.package;
+  configDirectory = pkgs.runCommand "freeswitch-config-d" { } ''
+    mkdir -p $out
+    cp -r ${cfg.configTemplate}/* $out
+    chmod -R +w $out
+    ${concatStringsSep "\n" (mapAttrsToList (fileName: filePath: ''
+      mkdir -p $out/$(dirname ${fileName})
+      cp ${filePath} $out/${fileName}
+    '') cfg.configDir)}
+    chmod -R -w $out
```

Comment:
This can be removed, since all of the Nix store is readonly by default anyways.

Reply:
Done.